jekyll/_cci2/github-app-trigger-options.adoc

```diff
@@ -0,0 +1,83 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= GitHub App trigger options
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A guide to the options available for triggering GitHub App pipelines
+:icons: font
+:experimental:
+
+When you create a trigger for a GitHub App pipeline, you can choose from a set of trigger options that correspond to link:https://docs.github.com/en/webhooks/webhook-events-and-payloads[GitHub App events].
+
+== Introduction
+
+When you set up a project in CircleCI you will also set up your first pipeline and trigger. This first "default" trigger is an "all push" trigger, which corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event]. You can also set up additional triggers for your project during setup, or later in your menu:Project settings[Triggers].
+
+Once your project is set up you can create additional pipelines and triggers (menu:Project settings[Triggers]). You can configure your triggers to initiate pipelines on a variety of events, including non-push events, for example, a pull request being opened, or refined push events, for example, only pushes to a default branch, or pushes to a tag.
+
+== Scope of push event support
+
+* Pipelines are not triggered on branch or tag deletion push events, even when you have the "All push" trigger option selected.
+* Pipelines are not triggered when a new branch is created if there are no new commits on the branch.
+
```

Comment:
I wonder if it's worth transforming this into a "Note". 
I say this because this has actually always been the behaviour of circleci OAuth historically , and afaik noone has ever noticed or complained .

Could be something like:
"Note: regardless of the chosen trigger option, CircleCI never runs pipelines on branch or tag deletion push events, not when a new branch is created if there are no new commits on the branch